policy/modules/system/modutils.te

```diff
@@ -44,6 +44,7 @@ allow kmod_t self:rawip_socket create_socket_perms;
 # Read module config and dependency information
 list_dirs_pattern(kmod_t, modules_conf_t, modules_conf_t)
 read_files_pattern(kmod_t, modules_conf_t, modules_conf_t)
+allow kmod_t modules_dep_t:file map;
```

Comment:
Please change line 46 to `mmap_read_files_pattern` instead.

Reply:
line 46 is about `modules_conf_t`.
BTW: is there any reason why the permission `map` is not included in `manage_file_perms`?

Reply:
My misake. Map is specifically controlled because it is not revocable.

Reply:
> Map is specifically controlled because it is not revocable.

Somewhat unrelated to this pull-request, as far as I know `map` isn't a special permission at all.
Are you referring to changes to the label of the file while it is opened, where `read` are checked again against the new label, but access to mapped areas is not ?

Reply:
> > Map is specifically controlled because it is not revocable.
> 
> Somewhat unrelated to this pull-request, as far as I know `map` isn't a special permission at all.
> Are you referring to changes to the label of the file while it is opened, where `read` are checked again against the new label, but access to mapped areas is not ?

Yes.